tests/Unit/CompanyBanTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Tests\Unit;
+
+use Tests\TestCase;
+use TruckersMP\APIClient\Collections\Company\BanCollection;
+use TruckersMP\APIClient\Models\CompanyBan;
+
+class CompanyBanTest extends TestCase
+{
+    /**
+     * The ID of the company to use in the tests.
+     */
+    private const TEST_COMPANY = 2;
+
+    /** @test */
+    public function it_can_get_all_the_bans()
+    {
+        $bans = $this->companyBans(self::TEST_COMPANY);
+
+        $this->assertInstanceOf(BanCollection::class, $bans);
+
+        if ($bans->count() > 0) {
+            $ban = $bans[0];
+
+            $this->assertInstanceOf(CompanyBan::class, $ban);
+        }
+    }
```

Comment:
This is a perfect example of why we should push for #30 first.

Reply:
@ShawnCZek Do you want to keep this PR open until #30 has been implemented? Because this is also how it's already done in other tests:
![image](https://user-images.githubusercontent.com/10339564/141458966-d5983f87-583a-4a50-9e56-4d1e3bc6b860.png)

If this only was used in this PR I would agree, however, since it's used in quite a few other tests as well, it might make more sense to fix all these issues in the PR for #30, when ready.

Reply:
You are right. And testing one instance here is not really possible (bans do not last forever).
This should indeed be changed after the change is done not to block this request.